docs/tutorials/blog.md

```diff
@@ -485,37 +457,31 @@ Update the `app/models/post.server.ts` file:
 ```tsx filename=app/models/post.server.ts lines=[3,9-11]
 import { prisma } from "~/db.server";
 
-export type { Post } from "@prisma/client";
-
 export async function getPosts() {
   return prisma.post.findMany();
 }
 
-export async function getPost(slug: string) {
+export async function getPost(slug?: string) {
   return prisma.post.findUnique({ where: { slug } });
 }
 ```
 
-If you see a TypeScript warning, such as `TS2305: Module '"@prisma/client"' has no exported member 'Post'.`, you may need to restart your editor.
```

Comment:
This could be used later when actually using an imported Prisma type in the post create action